crypto/keys/multisig/multisig.go

```diff
@@ -128,3 +139,28 @@ func (m *LegacyAminoPubKey) GetThreshold() uint {
 func (m *LegacyAminoPubKey) Type() string {
 	return "PubKeyMultisigThreshold"
 }
+
+// UnpackInterfaces implements UnpackInterfacesMessage.UnpackInterfaces
+func (m *LegacyAminoPubKey) UnpackInterfaces(unpacker types.AnyUnpacker) error {
+	for _, any := range m.PubKeys {
+		var pk tmcrypto.PubKey
+		err := unpacker.UnpackAny(any, &pk)
```

Comment:
Should we unpackAny into a `tmcrypto.PubKey` or a `cryptotypes.PubKey`? Or it doesn't matter?

Reply:
I believe we can use `cryptotypes.PubKey` instead